jsonschema-core.md

```diff
@@ -2079,7 +2079,7 @@ consumers will need to properly interpret validation results.
 ### Format
 
 JSON Schema output is defined using the JSON Schema data instance model as
-described in section 4.2.1. Implementations MAY deviate from this as supported
+described in [Instance Data Model](#data-model). Implementations MAY deviate from this as supported
```

Comment:
The markdown linter that's setup will flag this line as too long. It might be a good idea to setup a Github Action to check the linting results sooner than later as we start getting into the changes. Or, we need to be disciplined about running the build locally (`npm run build`) when we make changes.

Reply:
I actually prefer not maintaining line length.  I see the benefit in having new lines on new sentences, but not really in the middle of sentences.

Reply:
I edit in vim where very long lines are difficult to work with and reflowing the text is simple. I strongly prefer maintaining line lengths, but I can deal with it if necessary.

By default the linter plugin is designed to enforce something called the "Markdown Style Guide". I don't know how widely adopted that style guide is, but I like the idea of adopting a common standard. It appears that the linter can be customized to some degree. We can probably remove the line length rule if we decided we want to, but I haven't looked into how easy or difficult that might be.

Reply:
I don't mind adjusting this for now, but I think we should have this bigger conversation.  I think there are practical reasons (not just my preferred editor) to use long lines and word wrapping.